### ISSUE TITLE
Code generation producing model-specific InputStream rather than java.io.InputStream

### ISSUE THREAD

**Issue by brucej72** [micronaut-projects/micronaut-openapi#1230]
### Expected Behavior

When generating classes for https://platform.reapit.cloud/docs/swagger/agencyCloud_swagger.json into package com.reapit.api and com.reapit.model I expect the generated class DocumentsAPI.java to have the method:
```
@Get("/documents/{id}/download")
    @Consumes({"application/octet-stream"})
    Mono<InputStream> getDocumentContentById()
```

### Actual Behaviour

The generated code has the incorrect package for InputStream. Instaead of being java.io.InputStream it is referencing com.reapit.model.InputStream which does not exist.
This seems to be because the endpoint is defined as:
`        "responses": {
          "200": {
            "description": "Success",
            "schema": {
              "format": "binary",
              "type": "string"
            }
          },`
which is correct per the OpenAPI spec, but the generator is not using java.io.InputStream for it.

### Steps To Reproduce

1. Following this example here: https://guides.micronaut.io/latest/micronaut-openapi-generator-client-maven-java.html
2. substituting curl https://platform.reapit.cloud/docs/swagger/agencyCloud_swagger.json -o src/openapi/reapit.json in step 4.1 and
3. `<micronaut.openapi.definition>src/openapi/reapit.json</micronaut.openapi.definition>
    <micronaut.openapi.api.package.name>com.reapit.api</micronaut.openapi.api.package.name>
    <micronaut.openapi.model.package.name>com.reapit.model</micronaut.openapi.model.package.name>` in 4.2
4. mvn compile generates an error

### Environment Information

MacOS X 13.6
JDK 17.0.7

### Version

4.1.2

**Comment by brucej72** [comment]
This could be fixed by #1225 but I'm not sure

**Comment by altro3** [comment]
@brucej72 yes, it's already fixed. Wait next release
